### PR TITLE
[wpt/revlist] Add a flag to turn on debug logging

### PR DESCRIPTION
**Pull request by Hexcles**
./wpt revlist --verbose

### CODE REVIEW

**Comment by Hexcles** [comment]
cc @jgraham I looked for similar flags in other commands and found none (except `run` of course). I feel like this perhaps should be a global flag at the `wpt` front-end level?

File: tools/wpt/revlist.py [hunk 1]
```diff
@@ -106,11 +106,14 @@ def get_parser():
                         type=int,
                         help="maximum number of revisions to be returned by "
                              "the command")
+    parser.add_argument("--verbose", action="store_true", help="debug logging")
```
Comment (LukeZielinski) [inline tools/wpt/revlist.py]: Nit: also use "-v" for convenience, which seems common practice for other commands?
```suggestion
    parser.add_argument("-v", "--verbose", action="store_true", help="debug logging")
```

**Review by LukeZielinski — APPROVED** [review]

File: tools/wpt/revlist.py (hunk 1 above)
Comment (Hexcles) [reply tools/wpt/revlist.py]: I'm not sure if we want to leave `-v` for (a hypothetical) `--version` or `--verbose`. Which other wpt commands use `-v`? Or are you talking about CLI commands in general?

Comment (LukeZielinski) [reply tools/wpt/revlist.py]: Yeah fair point - just two spots in this codebase where it's used, so likely not enough to justify making this the common pattern:
https://github.com/web-platform-tests/wpt/search?q=%22-v%22%2C+%22--verbose%22+-path%3Atools%2Fthird_party&unscoped_q=%22-v%22%2C+%22--verbose%22+-path%3Atools%2Fthird_party

**Comment by jgraham** [comment]
I feel like adding flags like `--verbose` to wpt rather than the subcommands is confusing, because it's sort of unclear what the scope is i.e whether it's the subcommand or just the dispatch framework (and indeed for commands that *do* use mozlog, the scope would be "just the things that happen before we set up the mozlog logger").

**Pull request merged**